### ISSUE TITLE
[studio-ui] Forms engine xml serializer issue with video-S3-transcoding

### ISSUE THREAD

**Issue by jvega190** [craftercms/craftercms#4714]
## Describe the bug
When using the video-s30-transcoding + transcoded-video-picker, saving the form doesn't properly create the xml, and the video items are not saved into the form.
The xml serializer used by the forms-engine has an issue in the `recursiveRetrieveItemValues` method, exactly in this piece of code
```
output += '\t<' + fieldName + repeatAttr + '>';
```
due to the lack of a space between `fieldName` and `repeatAttr`.

## To Reproduce
1. Configure datasource and control to use media convert on a content type https://docs.craftercms.org/en/3.1/developers/cook-books/aws/upload-transcode-video.html
2. Upload a video using the added control.
3. Save form.
4. Open form again and check control.
5. See that the control shows 'undefined', and if you check the content, you'll see that there is no content related with the video uploaded.

## Expected behavior
Serializer should correclty generate the xml including the video-S3-transcoding content.

## Screenshots
{{If applicable, add screenshots to help explain your problem.}}

## Logs
{{If applicable, attach the logs/stack trace (use https://gist.github.com).}}

### Version
3.1.15-SNAPSHOT-df0db0

### OS
{{What OS did you use to produce the bug.}}

### Browser
{{What browser did you use to produce the bug.}}

## Additional context
{{Add any other context about the problem here.}}

**Comment by jvega190** [comment]
@yacdaniel assigning it to you for testing. Please ping me if you need any detail

**Comment by yacdaniel** [comment]
Verified fixed in `3.1.15-SNAPSHOT`

**Comment by yacdaniel** [comment]
Verified fixed in `4.0.0-SNAPSHOT` XML of page is saved properly
```
        <transcondedVideo_o>    <item>  <url remote="true">/remote-assets/s3/s3-default/studio-output-2/videos/file/refresh assets upload_file.mp4</url>
        </item> <item>  <url remote="true">/remote-assets/s3/s3-default/studio-output-2/videos/hls/refresh assets upload.m3u8</url>
        </item> <item>  <url remote="true">/remote-assets/s3/s3-default/studio-output-2/videos/dash/refresh assets upload.mpd</url>
        </item> <item>  <url remote="true">/remote-assets/s3/s3-default/studio-output-2/videos/smooth/refresh assets upload.ism</url>
        </item> <item>  <url remote="true">/remote-assets/s3/s3-default/studio-output-2/videos/cmaf/refresh assets upload.m3u8</url>
        </item> <item>  <url remote="true">/remote-assets/s3/s3-default/studio-output-2/videos/cmaf/refresh assets upload.mpd</url>
        </item></transcondedVideo_o>
```
